main.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import pygame
+
+from src.environment.environment import Environment
```

Comment:
Is there a reason why you put environment and agent inside subdirectories? It is not really a problem, but I find this line slightly strange to read with the 3 times `environment`.